test/library/gui/dwarfmode.lua

```diff
@@ -26,7 +26,7 @@ function test.enterSidebarMode()
     -- Simulate not being able to get to default from a screen via mocks. This
     -- failure can actually happen in-game in some situations, such as when
     -- naming a building with ctrl-N (no way to cancel changes).
-    mock.patch({{dfhack.gui, 'getFocusString', mock.func()},
+    mock.patch({{dfhack.gui, 'getFocusStrings', mock.func()},
```

Comment:
this test can be removed

Reply:
Tonight I was thinking "man, I really hope myk002 doesn't think this has been more trouble than it's worth..". That was 20 comments ago lmao. I super appreciate your help and hope my lack of knowledge here isn't _too_ annoying 😅

Reply:
not at all. you're doing great!

It would be nice to merge in the widespread focus string changes in a separate PR and leave this PR for just confirm.  That way I can work on focus string handling in ZScreens while you are finishing up confirm

Reply:
Thanks 😄.

>It would be nice to merge in the widespread focus string changes in a separate PR

Okay cool I can work on that. I'll focus my next efforts on getting the rest of the focus strings I was missing in and getting strictly focus string related stuff into a separate PR.

Then I'll try the suggestion about the detecting the trickier buttons. I wish every button just had a tooltip 😢